src/components/BottomBar/BottomBarTabView.vue

```diff
@@ -70,97 +76,33 @@ export default Vue.extend({
     };
   },
   props: {
-    bottomCourses: { type: Array as PropType<AppBottomBarCourse[]>, required: true },
-    seeMoreCourses: { type: Array as PropType<AppBottomBarCourse[]>, required: true },
-    bottomCourseFocus: { type: Number, required: true },
-    isExpanded: { type: Boolean, required: true },
     maxBottomBarTabs: { type: Number, required: true },
   },
 
   computed: {
-    seeMoreString() {
-      return 'See More';
+    bottomCourseFocus(): number {
+      const focus = immutableBottomBarState.bottomCourseFocus;
+      return focus < this.maxBottomBarTabs ? focus : this.maxBottomBarTabs - 1;
```

Comment:
I think the course focus may switch correctly to the right bottombartab but not the actual bottombarcourse content and title. When I select PE 1130, a course that's 3rd or 4th and minimize the screen width, shouldn't the course focus be on PAM 2100?

https://user-images.githubusercontent.com/54298311/108453811-37373780-7239-11eb-9bff-dfd15b35ac04.mov